fixieai/cli/agent/commands.py

```diff
@@ -167,14 +167,11 @@ def init_agent(handle, description, entry_point, more_info_url, requirement, lan
 
     is_typescript = language.lower() in ["typescript", "ts"]
 
-    try:
-        current_config = agent_config.load_config()
-    except FileNotFoundError:
-        current_config = agent_config.AgentConfig()
+    current_config = agent_config.AgentConfig()
```

Comment:
I realized we could simplify here, since the command no longer operates on a current config.